modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc

```diff
@@ -39,21 +36,4 @@ platform:
     ingressVIPs:
       - <wildcard_ipv4>
       - <wildcard_ipv6>
-----
-
-ifdef::vSphere[]
-[IMPORTANT]
-====
-You can configure dual-stack networking on a single interface only.
```

Comment:
I thought we did that because we weren't properly setting all of the node IPs, which should be fixed in 4.14. We don't limit dual stack to a single interface on baremetal and VSphere should work the same now.

Reply:
Uh I think the way of thinking here was different... So I start from https://docs.openshift.com/container-platform/4.13/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.html which say that there is a following limitation in OVN-K8s (platform-independent)

>For clusters configured for dual-stack networking, both IPv4 and IPv6 traffic must use the same network interface as the default gateway

From this, no matter what we do in the kubelet, OVN-K8s still needs to have default gateway on the same interface.

While I believe you could hack your way around to have default-gateway on NIC1 and then kubelet on NIC2, not sure this would be supported or even working correctly

Reply:
Ah, yes. I think what you're referring to is still true, but it's not specific to VSphere so I don't think that's what this was talking about.

Reply:
I think it was a misplaced `ifdef`.  The `ifdef` should have gone after this note, we want to keep this note for dualstack platforms, baremetal, vSphere, and ShiftStack [OSDOCS-5861](https://issues.redhat.com/browse/OSDOCS-5861)

This modules in included from `installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc` as well.

Reply:
I'm not sure about that. I believe we did deliberately limit VSphere dual stack deployments in 4.13 to single interface nodes because of the workaround we had to do to get it working in that release.

However, the fact that there is this much confusion about it makes me think the comment needs re-wording anyway. I'd be fine with leaving it if we change the wording to something like "In dual stack deployments both v4 and v6 node IPs must be assigned to the same interface." which will still be accurate and is much clearer IMHO.

Reply:
There was confusion about this since the first time we added next platform to supported by dual-stack because `In dual stack deployments both v4 and v6 node IPs must be assigned to the same interface` does not apply only to vSphere, it's also true in other platforms